ro_crate_ruby.gemspec

```diff
@@ -1,3 +1,6 @@
+lib = File.expand_path('../lib', __FILE__)
```

Comment:
Does this replace the need for the `require_relative` in lib/ro_crate_ruby.rb ?

Reply:
so, yes, because the lib folder is in the load_path you can replace `require_relative` with `require`, e.g. 
```
require_relative './ro_crate/model/entity'
```
with:
```
require 'ro_crate/model/entity'
```

- I'll update my PR with this change